src/com/cft/contactmerge/MatchMaker.java

```diff
@@ -0,0 +1,23 @@
+package com.cft.contactmerge;
```

Comment:
It might not be a bad idea to have a purpose description at the top of the class for what the class does (On all our classes). Example:
```
/*
 * This class is generally responsible for xyz.
 */
```
... Since we are both bad at naming stuff :)

Reply:
Pending. I was going to make this change right away but I messed up something in my branch and so need to merge the original change before I can make modification or I'll lose some other changes I made today.